pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [tool.poetry]
-name = "python3-saml"
+name = "maykin-python3-saml"
 version = "1.16.0"
 description = "Saml Python Toolkit. Add SAML support to your Python software using this library"
 license = "Apache-2.0"
-authors = ["SAML-Toolkits <contact@iamdigitalservices.com>"]
+authors = [
+    "SAML-Toolkits <contact@iamdigitalservices.com>",
+    "Maykin Media <support@maykinmedia.nl>",
+]
 maintainers = ["Sixto Martin <sixto.martin.garcia@gmail.com>"]
 readme = "README.md"
-homepage = "https://saml.info"
-repository = "https://github.com/SAML-Toolkits/python3-saml"
+homepage = "https://github.com/maykinmedia/python3-saml@maykin"
+repository = "https://github.com/maykinmedia/python3-saml@maykin"
```

Comment:
404s for both these URLs

Reply:
removed the @ - that branch is set as default branch anyway.